custom/icds_reports/templates/icds_reports/icds_app/mobile/progress-report.directive.html

```diff
@@ -0,0 +1,70 @@
+{% load staticfiles %}
+<div cg-busy="[$ctrl.myPromise, $rootScope.locationPromise]">
+  <div class="app-bg h-100vh overflow-hidden" ng-show="!$ctrl.filtersOpen">
+      <div class="top-section p-3" ng-style="{ 'paddingBottom' : (!$ctrl.report) ? '25vh' : '1rem' }">
+          {% include 'icds_reports/icds_app/mobile/partials/mobile-heading.html' %}
+          <p class="font-weight-600 font-montserrat font-16 text-white mb-0" ng-bind="$ctrl.title"></p>
+      </div>
+      <div class="section-list" ng-if="!$ctrl.report">
+          <div class="fact-sheet-nav-nutrition fact-sheet-sections col-xs-11" ng-click="$ctrl.goToReport('maternal_and_child_nutrition')">
+              <img src="{% static 'icds_reports/mobile/images/motherchildnutrition.png'%}" class="pr-3 section-images">
+              <p class="mb-0 font-montserrat font-weight-600 font-16">Maternal And Child Nutrition</p>
+          </div>
```

Comment:
Yes, I tried that first thing.. but I could not dynamically pass images from data structure. This is stopping me: https://stackoverflow.com/questions/35621030/unable-to-provide-image-names-dynamically-to-static-in-angularjs

Reply:
interesting. why is that a blocker?

Two possible options:
1. in other places we have just hard-coded the path since we don't ever change it. it's not ideal, but you can replace the template tag with just `/static/` and it still works. see - e.g. https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/static/css/icds_dashboard.less#L386
2. Can we define the data structure in python and pass it through as a constant? Then we can generate the URLs properly, like we do for program summary nav: https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/dashboard_utils.py#L56-L70

Reply:
I tried extracting datastructure into the controller, but did not consider the option of extracting it to python file (although we did it earlier). I will make that change